lunabot_control/scripts/excavation_controller.py

```diff
@@ -115,14 +115,19 @@ def publish_effort(self):
         effort_msg.lead_screw = self.constrain(
             self.lead_screw_voltage, max_percent=self.max_lead_screw_percent
         )
-        effort_msg.excavate = self.constrain(
+        effort_msg.excavate = self.constrain_RPM(
             self.exc_voltage, max_percent=self.max_exc_percent
```

Comment:
It would probably be a good idea to change exc_voltage to exc_rpm or something similar, unless I'm misunderstanding what the varaible does.

Reply:
Updated in fix threshold commit